backend/backend.py

```diff
@@ -79,13 +94,23 @@ def on_loop(self) -> None:
         if time() > self.prev_recording_status_time + self.recording_status_interval:
             self.send_recording_status()
 
+        if time() > self.prev_camera_is_online_time + self.camera_is_online_interval:
+            self.send_camera_is_online()
+
     @abstractmethod
     def _on_loop(self) -> None:
-        """ Implemented by overlays to perform any neccessary operations that
+        """ Implemented by overlays to perform any necessary operations that
             should be performed on a regular period. This may involve updating
             the display, which may be blocking. Should not be called outside
             of the Backend class. """
 
+    def send_camera_is_online(self) -> None:
```

Comment:
Could you move this to be under the `send_recording_error` function? Just so we have all the `send_*` functions grouped together, seems neater.

Reply:
Done
4358d96